modules/rostests/apitests/ws2_32/WSAAsync.c

```diff
@@ -36,6 +37,8 @@ START_TEST(WSAAsync)
     struct fd_set select_efds;
     struct timeval timeval;
     BOOL ConnectSent = FALSE;
+    unsigned int Addr_con_locLoopCount = 0,
+                 ServerSocketLoopCount = 0;
```

Comment:
This is not in line with the style around it.

Reply:
Which style exactly?
Maybe, just tell what you want...

Reply:
Variable declarations combined are on one line (the int's above),
Variable declarations broken over multiple lines all have their own type per line (so not combined with a comma).

Reply:
Well, not true at lines 18-21...
Although I don't really care either way.